packages/eslint-plugin/tests/rules/component-max-inline-declarations.test.ts

```diff
@@ -14,103 +14,68 @@ import rule, {
 const ruleTester = new RuleTester({
   parser: '@typescript-eslint/parser',
 });
+const messageId: MessageIds = 'componentMaxInlineDeclarations';
 
 ruleTester.run(RULE_NAME, rule, {
   valid: [
-    // should succeed if the number of lines does not exceeds the default lines limit (template)
+    // should succeed if the number of the template lines does not exceeds the default lines limit
     `
     @Component({
       template: '<div>just one line template</div>'
     })
     class Test {}
     `,
-    {
-      // should succeed if a negative limit is used and the number of lines does not exceeds the default lines limit (template)
-      code: `
-      @Component({
-        template: '<div>first line</div>'
-      })
-      class Test {}
-      `,
-      options: [
-        {
-          template: -5,
-        },
-      ],
-    },
-    // should succeed if the number of lines does not exceeds the default lines limit (styles)
+    // should succeed if the number of the styles lines does not exceeds the default lines limit
     `
     @Component({
       styles: ['div { display: none; }']
     })
     class Test {}
     `,
-    {
-      // should succeed if a negative limit is used and the number of lines does not exceeds the default lines limit (styles)
-      code: `
-      @Component({
-        styles: ['div { display: none; }']
-      })
-      class Test {}
-      `,
-      options: [
-        {
-          styles: -5,
```

Comment:
There no longer seems to be any coverage for the case where the user provides an invalid negative number

Reply:
Yup, as I've said in the PR description, I moved the manual check we had to schema and thus we don't need to have tests for this anymore, as the configuration is now invalid. That said, I noticed that despite not being a correct configuration or thinking that someone has this configuration with a negative number, it can be considered a breaking change, right? In any case, I'm okay in dropping the first commit and open a follow-up PR to be possibly merged in v2 if that's the case. Let me know your thoughts so I can see what I can make here, thanks.

Reply:
Yeah I'll mark it as a breaking change, but we can do v2 soon so not a big deal.

I'm still not 100% sure on why you wouldn't want to have test coverage for invalid configuration? Surely that's a good thing to capture in tests so that it doesn't inadvertently get changed over time?

Reply:
Unless I'm missing something, I don't think we can test it, because as soon as we put it there, we receive the following error:

![Captura de Tela 2021-02-02 às 09 22 21](https://user-images.githubusercontent.com/11965907/106599690-2ea3e900-6538-11eb-946f-e66232802550.png)

That being said, can we move this part to a follow-up PR and then merge this? I want avoid blocking #304 fix with this.

Reply:
So you can't move it to the invalid cases (looks like you still have it in valid) and capture it there? That might be the case I just assumed invalid would also mean invalid configuration for the RuleTester...

Are you saying you will remove the breaking change? That sounds fine if so

Reply:
> So you can't move it to the invalid cases (looks like you still have it in valid) and capture it there? That might be the case I just assumed invalid would also mean invalid configuration for the RuleTester...

Let's move this discussion to a follow-up PR. I'll open it as soon as it gets merged.

> Are you saying you will remove the breaking change? That sounds fine if so

Yes, I already did it. Take a look please :)

Reply:
@JamesHenry Sorry to ping you, but have you ever have a chance to take another look here? 😄